tasks/test.py

```diff
@@ -123,6 +124,16 @@ def install_tools(ctx):
                     ctx.run(f"go install {tool}")
 
 
+@task
+def invoke_unit_tests(ctx):
+    """
+    Start unit testsuite of tasks
+    """
+    for _, _, files in os.walk("tasks/unit-tests/", topdown=False):
```

Comment:
Not sure on how the `topdown=False` is used here?

Reply:
Not necessary i'm removing it